src/EffectID.h

```diff
@@ -1,6 +1,8 @@
 #pragma once
 
-enum class EffectID : Int32
+// tolua_begin
+
+enum EffectID
```

Comment:
How exactly doesn't it work? What is your code?

Basically ToLua processes only lines that are either marked with `// tolua_export`, or between `// tolua_begin` and `// tolua_end`; those lines, when filtered, need to comprise a valid C++ source. Naturally, the tolua_begin and tolua_end lines need to be properly balanced and only 1 level deep.